tests/unit/test_eval_compute_rouge.py

```diff
@@ -0,0 +1,221 @@
+    pytest.approx(rouge_perl["rouge_l_precision"], RLP, abs=ABS_TOL)
+    pytest.approx(rouge_perl["rouge_l_f_score"], RLF, abs=ABS_TOL)
+
+
+def test_compute_rouge_python(rouge_test_data):
+    rouge_python = compute_rouge_python(
+        cand=rouge_test_data["candidates"], ref=rouge_test_data["references"]
+    )
+
+    pytest.approx(rouge_python["rouge-1"]["r"], R1R, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-1"]["p"], R1P, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-1"]["f"], R1F, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-2"]["r"], R2R, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-2"]["p"], R2P, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-2"]["f"], R2F, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-l"]["r"], RLR, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-l"]["p"], RLP, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-l"]["f"], RLF, abs=ABS_TOL)
+
+
+def test_compute_rouge_python_hi(rouge_test_data):
+    rouge_python = compute_rouge_python(
+        cand=rouge_test_data["candidates_hi"], ref=rouge_test_data["references_hi"], language="hi"
+    )
+
+    pytest.approx(rouge_python["rouge-1"]["r"], R1R_hi, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-1"]["p"], R1P_hi, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-1"]["f"], R1F_hi, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-2"]["r"], R2R_hi, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-2"]["p"], R2P_hi, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-2"]["f"], R2F_hi, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-l"]["r"], RLR_hi, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-l"]["p"], RLP_hi, abs=ABS_TOL)
+    pytest.approx(rouge_python["rouge-l"]["f"], RLF_hi, abs=ABS_TOL)
+
+
+def test_compute_rouge_perl_file(rouge_test_data, tmp):
+    tmp_cand_file = os.path.join(tmp, "cand.txt")
+    tmp_ref_file = os.path.join(tmp, "ref.txt")
+
+    with open(tmp_cand_file, "w") as f:
+        for s in rouge_test_data["candidates"]:
+            f.write(s + "\n")
+    with open(tmp_ref_file, "w") as f:
+        for s in rouge_test_data["references"]:
+            f.write(s + "\n")
+
+    rouge_perl = compute_rouge_perl(cand=tmp_cand_file, ref=tmp_ref_file, input_files=True)
+
+    pytest.approx(rouge_perl["rouge_1_recall"], R1R, abs=ABS_TOL)
+    pytest.approx(rouge_perl["rouge_1_precision"], R1P, abs=ABS_TOL)
+    pytest.approx(rouge_perl["rouge_1_f_score"], R1F, abs=ABS_TOL)
+    pytest.approx(rouge_perl["rouge_2_recall"], R2R, abs=ABS_TOL)
+    pytest.approx(rouge_perl["rouge_2_precision"], R2P, abs=ABS_TOL)
+    pytest.approx(rouge_perl["rouge_2_f_score"], R2F, abs=ABS_TOL)
+    pytest.approx(rouge_perl["rouge_l_recall"], RLR, abs=ABS_TOL)
+    pytest.approx(rouge_perl["rouge_l_precision"], RLP, abs=ABS_TOL)
+    pytest.approx(rouge_perl["rouge_l_f_score"], RLF, abs=ABS_TOL)
+
+
+def test_compute_rouge_python_file(rouge_test_data, tmp):
```

Comment:
use tmp fixture to make sure the tmp folder clean up after testing

Reply:
I'm using the tmp fixure here.